assets/scripts/ui.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-module.exports = true;
+const onSuccess = function(data){
+  console.log('data is: ' + JSON.stringify(data));
+  console.table(data.books);
+};
+
+const onSuccessInputBook = function(data){
```

Comment:
This is *almost* identical to the previous function. You could handle this with logic in the previous function and re-use it.